remake.yml

```diff
@@ -15,21 +15,29 @@ targets:
   all:
     depends: 3_visualize/out/figure_1.png
 
-  site_data:
-    command: download_nwis_data()
-  
-  1_fetch/out/site_info.csv:
-    command: nwis_site_info(fileout = '1_fetch/out/site_info.csv', site_data)
+  1_fetch/out/site_data/01427207.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01432160.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01435000.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01436690.csv:
+    command: download_nwis_site_data(target_name)
+  1_fetch/out/site_data/01466500.csv:
+    command: download_nwis_site_data(target_name)
+    
+  all_site_files:
+    command: hash_dir_files(directory = I('1_fetch/out/site_data'), dummy = I('2020-07-01'))
```

Comment:
I like the hashing, but:
1. Given that you have to maintain the list of `depends` with all of the site files anyway, why not just make them into arguments of the `hash_dir_files` function? It's not a huge difference in fragility, but if the directory contents ever went out of sync with the `depends` - for example, if you deleted one of the site targets in this remake.yml without remembering to delete the file from 1_fetch/out/site_data - then the `hash_dir_files` would no longer reflect what you intended.
2. What does the `dummy` argument contribute?

Reply:
Good point.  For the dummy variable, I was looking for a way to be able to rerun all the individual site data requests with one target, it wouldn't help in this case.  I guess the way to do that would just be to delete the csv files?

Reply:
Deleting the .csvs would work. You could call `scdel` to do the same thing (and delete the .remake targets as well)...but I can't think of a good reason in this case to prefer `scdel()` to `unlink()` or `rm`.

As an alternative use of the dummy variable, how about creating a new target called something like `retrieval_date` and having each of the `download_nwis_site_data` targets depend on `retrieval_date`?